src/test/suite/fixtures/recorded/languages/cpp/changeCallee3.yml

```diff
@@ -0,0 +1,23 @@
+languageId: cpp
+command:
+  version: 1
+  spokenForm: change callee
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: functionCallee, includeSiblings: false}
+initialState:
+  documentContents: Car carObj1("X5", 1999);
+  selections:
+    - anchor: {line: 0, character: 17}
+      active: {line: 0, character: 17}
+  marks: {}
+finalState:
+  documentContents: Car ("X5", 1999);
```

Comment:
We would probably like this to include the type, but I'm currently not sure how to implement that. The type is in the field declaration[type] but the rest of what we want is in declaration[declarator][declarator]. Simply taking declaration[declarator] would include the arguments list. I'm open to suggestions.